dashboard/assets/js/containers/product-info-tab.js

```diff
@@ -200,6 +200,64 @@ export class ProductInfo extends Component {
     );
   }
 
+  codeClimateProjectURL(githubURL) {
+    const a = document.createElement('a');
+    a.href = githubURL;
+    return 'https://codeclimate.com/github' + a.pathname;
```

Comment:
this doesn't work on IE. another minor issue is this is a utility function which only runs in a browser. might be better to consider using a regex or even a small js library like https://github.com/garycourt/uri-js